### ISSUE TITLE
Menus: the whole section needs improvements to work well with VoiceOver

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#12881]
Steps to reproduce:

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. In the WordPress app, go to Menus.
1. Swipe right to navigate through each item.
1. Try to edit a menu.

**Result:** headings and logically grouped items need to be grouped and several elements are not accessible, for example the down arrows are not accessible at all (cannot be tapped) while using VoiceOver—the whole section needs improvements to work well with VoiceOver. ([8s](https://cloudup.com/cbcHo37Af_a))

<img src="https://user-images.githubusercontent.com/1119271/68268181-69a42d80-0012-11ea-83d5-7303eba5828d.png" width="270" alt="menus">
<sup>Tested with WPiOS 13.5.0.3 (App Store version) on iPhone 6S iOS 13.2.</sup>

<p style="clear:both;"><sub>/hat tip @etoledom for the report, internal reference: p77Llu-bQ6-p2</sub></p>

**Comment by chipsnyder** [comment]
Re-testing this as part of some grounds keeping work. With the changes from @etoledom this has made some really great progress. The one thing I noticed while trying to use it is the selection reverts back a step when moving quickly through the VoiceOver flow. 

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. In the WordPress app, go to Menus.
1. Highlight a menu 
1. Double-tap to expand the areas option
1. While VoiceOver is still speaking the first couple of words ~"Menu area primary" swipe right to navigate to the next item
- **Notice** that the app reverts you back to the expanded menu option forcing you to swipe right again to select the option you were just on


<kbd>![Voice-Over 2020-10-06 11_00_14](https://user-images.githubusercontent.com/3384451/95219578-89644880-07c3-11eb-8475-755b98e998ef.gif)</kbd>

**Comment by mindgraffiti** [comment]
Confirmed fixed: https://d.pr/v/kGqprl. Tested on iPhone 11 Pro, iOS 13.7, WPiOS 16.1 (DEV).
